src/main/java/com/adobe/target/delivery/v1/model/MboxResponse.java

```diff
@@ -13,13 +13,18 @@
  */
 package com.adobe.target.delivery.v1.model;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
 
 /** Mbox response object. */
-@JsonInclude(JsonInclude.Include.NON_NULL)
 public class MboxResponse {
```

Comment:
@dcottingham please make sure that this is expected. I am not sure why we have `$_type`.